plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnectorConfiguration.java

```diff
@@ -243,6 +243,13 @@ public InitialAuthConfig getAuthConfigs() {
         return authConfigs;
     }
 
+    /**
+     * For testing purposes
```

Comment:
I use it in org/eclipse/che/plugin/docker/machine/integration/DockerProcessTest.java
So the package differs and I can't use package private methods.